src/shared/provider/Auth/index.tsx

```
@@ -0,0 +1,105 @@
+      setIsLoading(false)
+    }
+  }
+
+  const logout = async () => {
+    try {
+      // 로그아웃 API 호출 로직 (필요 시 구현)
+      // await fetch(`${API_BASE_URL}/v1/auth/logout`, { method: 'POST' })
+    } catch (error) {
+      console.error('Logout error:', error)
+    } finally {
+      setIsLoggedIn(false)
```

Biome

[error] This code is unreachable



(lint/correctness/noUnreachable)